site/content/in-dev/unreleased/metastores.md

```diff
@@ -24,12 +24,11 @@ weight: 700
 
 This page documents important configurations for connecting to a production database through [EclipseLink](https://eclipse.dev/eclipselink/).
 
-## Building Polaris with EclipseLink
+## Polaris EclipseLink
```

Comment:
FYI I'm still seeing a few occurrences of `-PeclipseLinkDeps=org.postgresql:postgresql:42.7.4` in the following pages:

getting-started/eclipselink/README.md
site/content/in-dev/unreleased/getting-started/quickstart.md
site/content/in-dev/unreleased/admin-tool.md

Reply:
Yup, I'm doing a new pass.

Reply:
It should be good now.